src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,17 @@
+// stories/MyButton.stories.tsx
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {MyButton} from './Button'
```

Comment:
Should use a more descriptive name for `MyButton`

Reply:
We can read more Docs in https://storybook.js.org/tutorials/intro-to-storybook/react-native/en/simple-component/

Reply:
Skip for now